tests/cycles.rs

```diff
@@ -21,6 +61,36 @@ impl ParallelDatabase for DatabaseImpl {
     }
 }
 
+impl Default for DatabaseImpl {
+    fn default() -> Self {
+        let mut res = DatabaseImpl {
+            storage: salsa::Storage::default(),
+        };
+
+        // Default configuration:
+        //
+        //     A --> B <-- C
+        //     ^     |
+        //     +-----+
+
+        res.set_a_invokes(CycleQuery::B);
+        res.set_b_invokes(CycleQuery::A);
+        res.set_c_invokes(CycleQuery::B);
+        res
```

Comment:
I'd maybe duplicate this setup in every test? When writing a new test, it took me a while to realize why I observe revisons higher than I expect.

Reply:
Yeah, good idea.